src/components/index.d.ts

```diff
@@ -325,7 +325,17 @@ export declare const CloudinaryImage: {
 }
 export declare const filePath: (publicId: any) => any
 
-export { COLORS } from './Colors'
+export declare const COLORS: {
```

Comment:
my IDE was not finding COLORS with the previous export 🤔 idk why, but this helps.

Reply:
Is fixing the IDE the only reason you made this change @WesleyDRobinson? 
My concern is anyone updating `Colors` in the future will have to know to manually change the definition here as well. That's probably okay if we're not building in new colors that often but maybe we should add a comment to `Colors.js` explaining the declaration here.

Reply:
that's a good point. definitely strange behavior on my end, and I assumed for others also. i retried the export from Colors.js (ran prettier) and that helps so far.